lib/preprocess/ParameterHandler.ts

```diff
@@ -41,6 +41,20 @@ export class ParameterHandler {
     configElement: Resource,
     genericsContext: GenericsContext,
   ): Resource | undefined {
+    // Use the override value if there is one
+    const overrideTerm = this.objectLoader.contextResolved.expandTerm('oo:override');
```

Comment:
I'm wondering if this is still needed here?
Aren't we doing all overriding in the preprocessing step now?

Reply:
Ah yea, I still have to remove the previous solution.